packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -30,6 +30,9 @@ export abstract class TableColumn<
     @attr({ attribute: 'sort-direction' })
     public sortDirection: TableColumnSortDirection = TableColumnSortDirection.none;
 
+    @attr({ attribute: 'sorting-disabled' })
```

Comment:
Feedback from Milan on the HLD: if `sortingDisabled` is `true` then `currentSortDirection` / `currentSortIndex` should resolve to `undefined`. I'll resolve this comment once I update the PR with that change.

Reply:
Updated